app/uk/gov/hmrc/soletraderidentificationfrontend/models/BusinessVerificationStatus.scala

```diff
@@ -40,20 +47,23 @@ object BusinessVerificationStatus {
       val businessVerificationStatusString = businessVerificationStatus match {
         case BusinessVerificationPass => BusinessVerificationPassKey
         case BusinessVerificationFail => BusinessVerificationFailKey
-        case BusinessVerificationUnchallenged => BusinessVerificationUnchallengedKey
+        case BusinessVerificationNotEnoughInformationToChallenge => BusinessVerificationNotEnoughInfoToChallengeKey
+        case BusinessVerificationNotEnoughInformationToCallBV => BusinessVerificationNotEnoughInfoToCallBVKey
         case SaEnrolled => BusinessVerificationSaEnrolledKey
+        case BusinessVerificationUnchallenged => BusinessVerificationUnchallengedKey
       }
 
       Json.obj(BusinessVerificationStatusKey -> businessVerificationStatusString)
     }
 
     override def reads(json: JsValue): JsResult[BusinessVerificationStatus] =
       (json \ BusinessVerificationStatusKey).validate[String].collect(JsonValidationError("Invalid business validation state")) {
-        case BusinessVerificationPassKey => BusinessVerificationPass
-        case BusinessVerificationFailKey => BusinessVerificationFail
-        case BusinessVerificationUnchallengedKey => BusinessVerificationUnchallenged
-        case BusinessVerificationSaEnrolledKey => SaEnrolled
+        case `BusinessVerificationPassKey` => BusinessVerificationPass
```

Comment:
I think you only need the ` characters around the key names if they start with a lower case b. Probably isn't doing any harm though.